device/dell/x86_64-dell_s6100_c2538-r0/Force10-S6100/pg_profile_lookup.ini

```diff
@@ -1,17 +1,17 @@
 # PG lossless profiles.
 # speed cable size    xon  xoff threshold xon_offset
-  10000  5m   1248    2288 35776  -4        2288
-  25000  5m   1248    2288 53248  -4        2288
-  40000  5m   1248    2288 66560  -4        2288
-  50000  5m   1248    2288 79872  -4        2288
- 100000  5m   1248    2288 165568 -4        2288
-  10000  40m  1248    2288 37024  -4        2288
-  25000  40m  1248    2288 56160  -4        2288
-  40000  40m  1248    2288 71552  -4        2288
-  50000  40m  1248    2288 85696  -4        2288
- 100000  40m  1248    2288 177632 -4        2288
-  10000  300m 1248    2288 46176  -4        2288
-  25000  300m 1248    2288 79040  -4        2288
-  40000  300m 1248    2288 108160 -4        2288
-  50000  300m 1248    2288 131456 -4        2288
- 100000  300m 1248    2288 268736 -4        2288
+  10000  5m   1248    2288 35776  -3        2288
+  25000  5m   1248    2288 53248  -3        2288
+  40000  5m   1248    2288 66560  -3        2288
+  50000  5m   1248    2288 79872  -3        2288
```

Comment:
@wendani 50G speed numbers are not correct. it should be 90272.

Reply:
@vharish02, can you create a separate PR to propose the correct change?

Reply:
@wendani Yes I can create, but the number in 201803 branch is correct link given below, I am not sure why your PR numbers are not reflecting correctly, that's the reason I am confused, because the PR is forward porting the S6100 from 201803 to master.
https://github.com/Azure/sonic-buildimage/blob/201803/device/dell/x86_64-dell_s6100_c2538-r0/Force10-S6100/pg_profile_lookup.ini